src/main.cpp

```diff
@@ -37,10 +39,12 @@ void testBoardComponents(){
 }
 
 void setup() {
-  Serial.begin(Properties::BAUD_RATE);  //Begin serial communication
-  testBoardComponents();
+  delay(5000);
+  Serial.begin(Properties::BAUD_RATE);
+  //testBoardComponents();
```

Comment:
Was turning this off in main intentional? If there are circumstances where we don't want testing to take place then we should wrap this with an if-check and put another entry in the Properties class with a name something like "TEST_COMPONENTS" or something.